GiniSDK Example/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -84,6 +84,11 @@
       "idiom" : "ipad",
       "size" : "83.5x83.5",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
+      "size" : "1024x1024",
+      "scale" : "1x"
```

Comment:
What is this change? I assume it was just because you opened the Assets file?

Reply:
Xcode did it automatically, I didn't want it for this PR but I think it is not a big deal.